test/mobile/nnc/test_aot_compile.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+TORCH_INSTALL_DIR=$(python -c "import site; print(site.getsitepackages()[0])")/torch
+TORCH_BIN_DIR="$TORCH_INSTALL_DIR"/bin
+CURRENT_DIR="$( cd "$( dirname "$0" )" >/dev/null 2>&1 && pwd )"
+
+MODEL=aot_test_model.pt
+COMPILED_MODEL=aot_test_model.compiled.pt
+COMPILED_CODE=aot_test_model.compiled.ll
+
+TMP_DIR=$(mktemp -d -t build_XXX)
```

Comment:
nit: build_XXX -> build_mobile_nnc

Anything starts with "build" under the pytorch src folder will be ignored by git: https://github.com/pytorch/pytorch/blob/master/.gitignore#L212

This way it should not complain about leaving local changes when the test exits.
"mktemp" will create a folder at /tmp, which won't cause local changes under pytorch src folder either, with or without the "build_" prefix.